src/entity/src/user.js

```diff
@@ -498,7 +499,13 @@ export default class User {
       client: this.client
     });
 
-    return request.execute()
+    let prm = Promise.resolve();
+
+    if (LiveServiceFacade.isInitialized()) {
+      prm = LiveServiceFacade.unregister();
```

Comment:
We do something similar for `User.logout()`. You can call this API even when there is not an active user and because the end result is the same. I would prefer to see where you call `LiveServiceFacade.unregister()` without worrying about calling `LiveServiceFacade.isInitialized()`.